### PR TITLE
Send losing bidders an email

### PR DESCRIPTION
**Pull request by jessieay**
* Closes https://github.com/18F/micropurchase/issues/421
* Use Mandrill for SMTP
* Use letter opener to intercept emails in development: https://github.com/ryanb/letter_opener
* Use clockwork to run job once a day at 3am UTC https://github.com/tomykaira/clockwork

This is what the email looks like:

screen shot 2016-04-22 at 4 47 38 pm

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,5 @@
 MPT_3500_GITHUB_KEY="your-client-id"
 MPT_3500_GITHUB_SECRET="your-client-secret"
+SMTP_PASSWORD='your-mandrill-api-key'
+SMTP_USERNAME='your-mandrill-email'
+DEFAULT_URL_HOST='localhost:3000'
```
Comment (jessieay) [inline .env.example]: these 2 env vars will be required for prod -- can someone help me create a fancy user-provided service for them?

Comment (harrisj) [inline .env.example]: Paging @adelevie?

Comment (jessieay) [inline .env.example]: done in this PR! Thanks @adelevie

**Comment by jessieay** [comment]
right now, something with the SMTP setup is causing this to not send. Will comment back when it's working

**Comment by jessieay** [comment]
This is blocked until I can verify staging and prod as sending domains in mandrill https://mandrill.zendesk.com/hc/en-us/articles/205582277-How-do-I-add-DNS-records-for-my-sending-domains-

**Comment by jessieay** [comment]
This is now working!

**Comment by jessieay** [comment]
I just checked out Code Climate failures - I disagree with all points (aside from class length one, but that is outside the scope of this PR). 

LMK if that's a problem! otherwise will open a PR to change some rules

**Comment by harrisj** [comment]
Okay, let's merge this!

**Pull request merged**